plugins/sentinelone/komand_sentinelone/util/api.py

```diff
@@ -1,10 +1,52 @@
 from json import dumps, loads
 from re import match
 from insightconnect_plugin_runtime.exceptions import PluginException
+from insightconnect_plugin_runtime.helper import clean_list, clean_dict
 
 import requests
 
 
+default_array = [
+    "computerMemberOf",
+    "lastUserMemberOf",
+    "locations",
+    "networkInterfaces",
+    "inet",
+    "inet6",
+    "userActionsNeeded",
+]
+
+
+def clean(obj):
+    """
+    Returns a new but cleaned JSON object.
+
+    * Recursively iterates through the collection
+    * None type values are removed
+    * Empty string values are removed
+
+    This function is designed so we only return useful data
+    """
+
+    cleaned = clean_list(obj) if isinstance(obj, list) else clean_dict(obj)
+
+    # The only *real* difference here is how we have to iterate through these different collection types
+    if isinstance(cleaned, list):
+        for key, value in enumerate(cleaned):
+            if isinstance(value, list) or isinstance(value, dict):  # pylint: disable=consider-merging-isinstance
+                cleaned[key] = clean(value)
+            if key in default_array and (value is None or value == "None"):
```

Comment:
This code is checking if theres a key matching in default_array, but the keys we're searching for here is in a list so the keys should always be 0, 1, 2, 3 etc and would never match any values in default_array

Reply:
We remove checking this key when cleaned is a list.